bin/PythonApps/entitlements.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>com.apple.security.cs.allow-exec</key>
+    <true/>
+    <key>com.apple.security.files.all-access</key>
+    <true/>
```

Comment:
And I think I needed these two.

allow-exec to run energyplus. And files.all-access so I can access energyplus outside of the .app directory.

I haven't really tried to remove them, it was a lengthy process already.